main.pm

```diff
@@ -624,6 +630,17 @@ else {
         }
         loadtest "installation/first_boot.pm";
     }
+    elsif (is_jeos) {
+        if (get_var("ISO_MAXSIZE")) {
```

Comment:
I probably does not get full usage. But shouldn't you check if HDD_1 is present too? You are using it in the test.